<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="BASIC TESTS"
           tests="2"
           failures="1"
           errors="0"
           skipped="1"
           time="0.017">
  <properties />
            <testcase name="failIt" time="0.015" classname="testcases.FailingTestCases">
                <failure type="java.lang.AssertionError"
                  message="Test Case Failed"
              >
<![CDATA[java.lang.AssertionError: Test Case Failed
  org.testng.Assert.fail(Assert.java:96)
  testcases.FailingTestCases.failIt(FailingTestCases.java:10)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:568)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
  org.testng.SuiteRunner.run(SuiteRunner.java:364)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
  org.testng.TestNG.runSuites(TestNG.java:1049)
  org.testng.TestNG.run(TestNG.java:1017)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
            <testcase name="skipIt" time="0.0" classname="testcases.FailingTestCases">
          <skipped />
    </testcase>
  
  
</testsuite>
